src/autoreply/eita.js

```diff
@@ -0,0 +1,17 @@
+const patterns = [/^eita/i];
+
+const answer = () => `levxycEita levxycEita levxycEita`;
```

Comment:
Use aspas simples ao invés de _template strings_ se não houver expressão a ser processada.

```suggestion
const answer = () => 'levxycEita levxycEita levxycEita';
```